config/config.go

```diff
@@ -565,3 +602,12 @@ func (c *Config) GetWarpQuorum(blockchainID ids.ID) (WarpQuorum, error) {
 	}
 	return WarpQuorum{}, errFailedToGetWarpQuorum
 }
+
+func (c *Config) GetSourceBlockchainAllowedAddresses(blockchainID ids.ID) []common.Address {
```

Comment:
It probably doesn't make a huge performance difference as long as there aren't a large number of supported source chains by an individual relayer, but should we consider creating a map here so we don't need to iterate `SourceBlockchains` each time? It looks like this is called each time the relayer sees a message, so I think it's probably worth a TODO if it's larger of a change than you'd like to make now.

Reply:
This method has been removed.